specifications/api/swagger.yaml

```diff
@@ -2358,6 +2374,25 @@ paths:
           - *parametersAddressState
         responses: *responsesListAddresses
 
+      put:
+        operationId: importAddresses
+        tags: ["Byron Addresses"]
+        summary: Import Addresses
+        description: |
+          <p align="right">status: <strong>stable</strong></p>
+
+          ⚠️  This endpoint is available for `random` wallets only. Any
```

Comment:
Maybe we should elaborate briefly on the difference between the endpoints such that people don't just discover the slow one, without realising that this one exists and is better suited for a large number of addresses.

Reply:
yes, it would be good, especially if the two survive..but as we talked today, the other one is rather redundant. We can decide tomorrow and if the other survive we will change it as you suggested

Reply:
> especially if the two survive..but as we talked today, the other one is rather redundant.

:grimacing: I'd argue that the new one is redundant ^.^ ... or more exactly, shall be made redundant once we fixed the issue with the database / checkpoint insertion. 

This should help: https://github.com/input-output-hk/cardano-wallet/pull/1663 although it'd need some more thoughts on the byron case.